docs/en/latest/plugins/batch-requests.md

```diff
@@ -109,6 +109,22 @@ Response is `Array` of [HttpResponse](#httpresponse).
 | body    | string  | http response body    |
 | headers | object  | http response headers |
 
+## How to specify custom uri
+
+We can change the default uri in the `plugin_attr` section of `conf/config.yaml`.
+
+| Name       | Type   | Default                      | Description                       |
```

Comment:
Could you also add column `Requirement` like http://apisix.apache.org/docs/apisix/plugins/proxy-rewrite/#attributes

Reply:
good suggestion